### ISSUE TITLE
Writing mixed-mode assemblies is not supported

### ISSUE THREAD

**Issue by kuhnboy** [brutaldev/StrongNameSigner#72]
Running into this error with mixed-mode assemblies with 3.1.0.

```
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error : NotSupportedException: Writing mixed-mode assemblies is not supported
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error :    at Mono.Cecil.ModuleWriter.Write(ModuleDefinition module, Disposable`1 stream, WriterParameters parameters)
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error :    at Mono.Cecil.ModuleWriter.WriteModule(ModuleDefinition module, Disposable`1 stream, WriterParameters parameters)
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error :    at Mono.Cecil.ModuleDefinition.Write(String fileName, WriterParameters parameters)
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error :    at Brutal.Dev.StrongNameSigner.AssemblyInfo.Save(String assemblyPath, Byte[] keyPair)
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error :    at Brutal.Dev.StrongNameSigner.SigningHelper.SignAssemblies(IEnumerable`1 assemblyInputOutputPaths, String keyFilePath, String keyFilePassword, String[] probingPaths)
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error :    at Brutal.Dev.StrongNameSigner.AutomaticBuildTask.Execute()
1> <<redacted>>\packages\Brutal.Dev.StrongNameSigner.3.1.0\build\Brutal.Dev.StrongNameSigner.targets(6,5): error :
```

It is implied that this is fixed in 3.1.0 (https://github.com/brutaldev/StrongNameSigner/pull/67)

**Comment by brutaldev** [comment]
@kuhnboy Mono.Cecil is clearly not happy about the assembly you are using.
You will need to provide me with the assembly you are trying to sign that causes that exception so I can reproduce and attempt to fix it.

**Comment by kuhnboy** [comment]
Ok thanks. So the correct answer is:
- Skipping mixed mode assemblies was broken but should be fixed in 3.1.1.
- There is no support for mixed mode assembly signing due to Cecil not supporting it.

**Comment by brutaldev** [comment]
Fixed and deployed to NuGet: https://www.nuget.org/packages/Brutal.Dev.StrongNameSigner/3.1.1
